docs/source/yelpsoa_configs.rst

```diff
@@ -39,6 +39,10 @@ instance MAY have:
     left on device' errors if they attempt to exceed these limits, and then be
     unable to write any more data to disk.
 
+  * ``ulimit``: Dictionary of ulimit options that are passed to Docker. Defaults
```

Comment:
Example added.

The code generates zero or more --ulimit options that are passed to Docker. For example: `--ulimit nofile=1024:2048 --ulimit nice=20`.

I changed the config format in the most recent version of my branch to explicitly specify soft and hard limits (instead of implicitly specifying the hard limit in the limit value).